gfx_scene.h

```diff
@@ -175,13 +175,22 @@ GfxMetadata const &gfxSceneGetCameraMetadata(GfxScene scene, uint64_t camera_han
 //! Image object.
 //!
 
+enum GfxImageFlags
+{
+    kGfxImageFlags_None = 0,
+    kGfxImageFlags_AlphaChannel = 1,
+
+    kGfxImageFlags_Count
+};
```

Comment:
Bit same feedback than the previous enum, I think I'd rather see:
```
enum GfxImageFlag
{
    kGfxImageFlag_HasAlpha = 1 << 0
};
```
With the `uint32_t` flags being initialized to `0`.
What do you think?

Reply:
yep ok done that